webapp/packages/core-localization/src/locales/en.ts

```diff
@@ -101,6 +101,10 @@ export default [
   ['app_root_session_expired_title', 'Session expired'],
   ['app_root_session_expired_message', 'Session has expired. Do you want to reload?'],
   ['app_root_session_expired_reload', 'Reload'],
+  ['app_root_server_node_changed_title', 'Server node changed'],
+  ['app_root_server_node_changed_message', 'Server node changed. You need to reload the application.'],
+  ['app_root_server_node_changed_reload', 'Reload'],
```

Comment:
Reload and Cancel language tokens are propably already exists in the codebase.

Reply:
Updated this